src/schemas.js

```diff
@@ -171,3 +170,247 @@ export const atomicSwapCreateSchema = {
     optional: true,
   },
 };
+
+export const txBuildSchema = {
+  outputs: {
+    in: ['body'],
+    errorMessage: 'Invalid outputs array',
+    isArray: true,
+    notEmpty: true,
+    optional: false,
+  },
+  'outputs.*.address': {
+    in: ['body'],
+    errorMessage: 'Invalid address',
+    isString: true,
+    optional: true,
+  },
+  'outputs.*.value': {
+    in: ['body'],
+    errorMessage: 'Invalid value',
+    isInt: {
+      options: {
+        min: 1,
+      },
+    },
+    toInt: true,
+    optional: true,
+  },
+  'outputs.*.token': {
+    in: ['body'],
+    errorMessage: 'Invalid token uid',
+    isString: true,
+    optional: true,
+    custom: {
+      // Test if token uid is a 64 character hex string or 00
+      options: value => /^(00|[0-9a-fA-F]{64})$/.test(value),
+    },
+  },
```

Comment:
Should we have `outputs.*.data` here as well? And `outputs.*.type`?

Reply:
Maybe we could have a validation chain for data outputs vs. funds outputs but using the custom validator seemed better.

We could add this validation chain, what do you think?

Reply:
For me it makes sense because we accept data outputs here, right?

Reply:
Hummm ok